src/page-multi-facility/FacilityChart.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+
+import { getFacilities } from "../database";
+import Colors, { MarkColors as markColors } from "../design-system/Colors";
+import Loading from "../design-system/Loading";
+import CurveChartContainer from "../impact-dashboard/CurveChartContainer";
+import { EpidemicModelProvider } from "../impact-dashboard/EpidemicModelContext";
+import useModel from "../impact-dashboard/useModel";
+import {
+  getNewestRt,
+  getRtDataForFacility,
+  isRtData,
+} from "../infection-model/rt";
+import { useLocaleDataState } from "../locale-data-context";
+import { initialPublicCurveToggles } from "./curveToggles";
+import FacilityRowRtValuePill from "./FacilityRowRtValuePill";
+import {
+  useChartDataFromProjectionData,
+  useProjectionData,
+} from "./projectionCurveHooks";
+import { Facility, RtValue } from "./types";
+
+const LoadingWrapper = styled.div`
+  color: ${Colors.opacityGray};
+  display: flex;
+  height: 45%;
+  align-content: center;
+`;
+
+interface FacilityChartProps {
+  facility: Facility | undefined;
+  facilityRtData: RtValue | undefined;
+}
+
+const FacilityChart: React.FC<{
+  scenarioId: string;
+  indexOfFacility: number;
+}> = ({ scenarioId, indexOfFacility }) => {
+  const [facility, setFacility] = useState<Facility | undefined>();
+  const [facilityRtData, setFacilityRtData] = useState<RtValue | undefined>();
+  const { data: localeDataSource } = useLocaleDataState();
+
+  useEffect(() => {
+    let mounted = true;
+    async function fetchFacility() {
+      const facilities = await getFacilities(scenarioId);
+      if (facilities) {
+        const facility = facilities[indexOfFacility];
+        const facilityRtData = await getRtDataForFacility(facility);
```

Comment:
I would recommend not "shadowing" the state variable names like this; it doesn't break anything per se but it makes things harder to read